packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarCell.tsx

```diff
@@ -124,17 +124,19 @@ export const DatepickerCalendarCell = compose<
     };
 
     const element = (
-      <ElementType
-        {...getA11yProps('root', {
-          className: classes.root,
-          onClick: handleClick,
-          ref,
-          disabled,
-          ...unhandledProps,
-        })}
-      >
-        {content}
-      </ElementType>
+      <td {...getA11yProps('tableCell', {})}>
```

Comment:
I'm not 100% sure on that one - please @layershifter could you take a look at that?

Reply:
Switched to slots. :)